test/framework/resource/k8s/pod/manager.go

```diff
@@ -239,5 +239,9 @@ func (d *defaultManager) getRestClientForPod(namespace string, name string) (res
 	if err != nil {
 		return nil, err
 	}
-	return apiutil.RESTClientForGVK(gkv, false, d.config, serializer.NewCodecFactory(d.k8sSchema))
+	client, err := rest.HTTPClientFor(d.config)
```

Comment:
I ended up deleting the equivalent of this function in VPC CNI PR: https://github.com/aws/amazon-vpc-cni-k8s/pull/2481, as it was causing issues, so you may want to check if the integration test that depends on this passes.

This function is only used for calling pod EXEC from integration tests, and you can copy the VPC CNI approach

Reply:
This function is right now not in the critical path, so we'll doing a follow up PR. We only call this function to test the data plane installation of the controller.  
Thanks for flagging